.env.example

```diff
@@ -0,0 +1,4 @@
+# Set these in .env.development
+AUTH0_DOMAIN=replace
+AUTH0_CLIENT_ID=replace
+AUTH0_CALLBACK=replace
```

Comment:
Set up Auth0
###  Set up an application on Auth0

> Lines2-4

If necessary, create an account and application on Auth0.

In the *settings* tab for your new Auth0 application, you'll need to set a few redirect urls. Those you see below are for `localhost:8000` for gatsby's development mode, and port `9000` for the built app when running via `gatsby serve`.

Finally for the domain, you'll need to set `AUTH0_DOMAIN`, `AUTH0_CLIENT_ID`, AND `AUTH0_CALLBACK`. The first two are at the top of your settings file, while the latter is what you set in the graphic below. We'll be using `http://localhost:8000/callback` for development.

Create a `.env.example` as above, and set the actual values in .env.development (they will also need to be set your production environment - `.env.production` when running `gatsby serve`).

![image](https://user-images.githubusercontent.com/9088720/58376692-fadc8780-7f3d-11e9-8493-d83872391a99.png)

[Next →](https://github.com/erikdstock/gatsby-starter-ts-auth0/pull/2/files#diff-fda05457e393bada716f508859bfc604R12)